config/csv/bases/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -277,10 +277,26 @@ spec:
           path: exporters[0].ipfix
           x-descriptors:
             - "urn:alm:descriptor:com.tectonic.ui:fieldDependency:exporters.type:IPFIX"
+        - displayName: IPFIX TargetHost
```

Comment:
okay this change did not help. Let me know what I am missing here

Reply:
I'm not sure why but these doesn't work anymore on arrays :disappointed: 
https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#20-fielddependency

Why not getting rid of these and simply keep:
```
          - description: additional optional exporters for custom consumption or storage.
            path: exporters
            x-descriptors:
              - 'urn:alm:descriptor:com.tectonic.ui:advanced'
          - displayName: Type
            path: 'exporters[0].type'
```

Reply:
![image](https://github.com/user-attachments/assets/2026e14e-863e-49f4-a993-e87025d3fe95)

Reply:
Lets see if this helps

Reply:
yeah so pushed your suggestion so looks as above.
But the flowcollector is not getting created due to error
<img width="886" alt="Screenshot 2024-11-27 at 7 15 10 PM" src="https://github.com/user-attachments/assets/e174efe4-5458-4a5e-a045-00e0e29cc84e">

Since the YAML view looks as below
<img width="278" alt="Screenshot 2024-11-27 at 7 15 30 PM" src="https://github.com/user-attachments/assets/cceecc1c-da7b-4585-a5e7-7e27bbac14c7">

Reply:
![abe-simpson-abe-simpson-cloud](https://github.com/user-attachments/assets/02cd476a-dd11-4abf-b12c-0af1f05739a7)

It seems like there is an issue with default values and this legacy auto generated form.
We may add default values to kafka address + topic and openTelemetry targetHost + targetPort to solve that.

@jotak thoughts on that ?